.github/workflows/release.yml

```diff
@@ -37,509 +37,500 @@ jobs:
-          repository: pulumi/scripts
-      - name: Unshallow clone for tags
-        run: git fetch --prune --unshallow --tags
-      - name: Install Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: ${{ env.GOVERSION }}
-      - name: Install pulumictl
-        uses: jaxxstorm/action-install-gh-release@v1.5.0
-        with:
-          repo: pulumi/pulumictl
-      - name: Install Pulumi CLI
-        uses: pulumi/action-install-pulumi-cli@v2
-      - if: github.event_name == 'pull_request'
-        name: Install Schema Tools
-        uses: jaxxstorm/action-install-gh-release@v1.5.0
-        with:
-          repo: mikhailshilkov/schema-tools
-      - name: Initialize submodules
-        run: make init_submodules
-      - name: Build codegen binaries
-        run: make codegen
-      - name: Build Schema
-        run: make generate_schema
-      - if: github.event_name == 'pull_request'
-        name: Check Schema is Valid
-        run: >-
-          echo 'SCHEMA_CHANGES<<EOF' >> $GITHUB_ENV
+    - name: Checkout Repo
+      uses: actions/checkout@v3
+      with:
+        lfs: true
+    - name: Checkout Scripts Repo
+      uses: actions/checkout@v3
+      with:
+        path: ci-scripts
+        repository: pulumi/scripts
+    - name: Unshallow clone for tags
+      run: git fetch --prune --unshallow --tags
+    - name: Install Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ${{ env.GOVERSION }}
+    - name: Install pulumictl
+      uses: jaxxstorm/action-install-gh-release@v1.5.0
+      with:
+        repo: pulumi/pulumictl
+    - name: Install Pulumi CLI
+      uses: pulumi/action-install-pulumi-cli@v2
+    - if: github.event_name == 'pull_request'
+      name: Install Schema Tools
+      uses: jaxxstorm/action-install-gh-release@v1.5.0
+      with:
+        repo: mikhailshilkov/schema-tools
+    - name: Initialize submodules
+      run: make init_submodules
+    - name: Build codegen binaries
+      run: make codegen
+    - name: Build Schema
+      run: make generate_schema
+    - if: github.event_name == 'pull_request'
+      name: Check Schema is Valid
+      run: >-
+        echo 'SCHEMA_CHANGES<<EOF' >> $GITHUB_ENV
 
-          schema-tools compare ${{ env.PROVIDER }} master --local-path=provider/cmd/pulumi-resource-${{ env.PROVIDER }}/schema.json >> $GITHUB_ENV
+        schema-tools compare ${{ env.PROVIDER }} master --local-path=provider/cmd/pulumi-resource-${{ env.PROVIDER }}/schema.json >> $GITHUB_ENV
 
-          echo 'EOF' >> $GITHUB_ENV
-      - if: github.event_name == 'pull_request'
-        name: Comment on PR with Details of Schema Check
-        uses: thollander/actions-comment-pull-request@v1
-        with:
-          message: |
-            ### Does the PR have any schema changes?
+        echo 'EOF' >> $GITHUB_ENV
+    - if: github.event_name == 'pull_request'
+      name: Comment on PR with Details of Schema Check
+      uses: thollander/actions-comment-pull-request@v2
```

Comment:
Changes should allow schema-tools to edit one comment instead of adding a new one like https://github.com/pulumi/pulumi-aws-native/pull/999